packages/moonstone/ExpandableItem/ExpandableItem.js

```diff
@@ -216,7 +216,15 @@ const ExpandableItemBase = kind({
 		 * @default false
 		 * @public
 		 */
-		spotlightDisabled: PropTypes.bool
+		spotlightDisabled: PropTypes.bool,
+
+		/**
+		 * Set voice control property to LabeledItem
+		 *
+		 * @type {Object}
+		 * @public
+		 */
+		voiceProps: PropTypes.object
```

Comment:
What are `voiceProps`? Are there a defined set which we should declare individually here?

Reply:
voiceProps is used to customize voice control properties of the ExpandableItem.
For example, if app developer want to set custom voice control label, they will declare it to ExpandableItem Component.
In current status, the props will be delegate to ContainerDiv(not LabeledItem)
About this modification, I guess we need more discussion, So, I will check again and make seperate ticket.(In this PR, I will remove voiceProps related code)